src/frontend/tsx/GamePanel.tsx

```diff
@@ -163,26 +164,30 @@ export function GamePanel () {
         {/* resign button */}
         <button
           className='option'
-          disabled={!hasGame}
+          disabled={!activeGame}
           onClick={resignOnClick}>
-          Resign</button>
+         Resign</button>
         {/* offer/accept draw button */}
         {displayGame.gotDrawOffer
           ? <button
             className='option'
+          /* XX: fails because visibility/visible don't exist on the
+           * HTMLButtonElement
+           *
+           * visibility={activeGame} */
             disabled={!hasGame}
```

Comment:
I think every usage in this part of `!hasGame` can be replaced by `!activeGame`:
- line `178`
- line `183`
- line `190`
- line `197`
- line `202`